api/v1/subschema/sauce.schema.json

```
@@ -73,6 +73,13 @@
             { "const": "team", "title": "Only accessible to people under the same root account as you." },
             { "const": "private", "title": "Only you (the owner) will be able to view assets and test results page." }
           ]
+        },
+        "launchOrder": {
+          "description": "Launch jobs based on the failure rate. Jobs with the highest failure rate launch first.",
```

That ought to be the description for fail rate.

As for launchOrder

```suggestion
          "description": "Control starting order of suites. The default is the order in which suites are written in the config file.",
```